docs/featurestore/featurestore.rst

```diff
@@ -1018,11 +846,11 @@ Instructions for installing Hopsworks (which comes with the feature store) on AW
 
 **How can I use Hopsworks Featurestore from Amazon Sagemaker?**
 
-Documentation for accessing Hopsworks Featurestore from Amazon Sagemaker is available here: `https://hopsworks.readthedocs.io/en/latest/user_guide/hopsworks/featurestore.html#connecting-from-amazon-sagemaker <https://hopsworks.readthedocs.io/en/latest/user_guide/hopsworks/featurestore.html#connecting-from-amazon-sagemaker>`_.
+Documentation for accessing Hopsworks Featurestore from Amazon Sagemaker is available here: Databricks_.
 
 **How can I use Hopsworks Featurestore from Databricks Platform?**
 
-Documentation for accessing Hopsworks Featurestore from Databricks Platform is available here: `https://hopsworks.readthedocs.io/en/latest/user_guide/hopsworks/featurestore.html#connecting-from-databricks-notebooks <https://hopsworks.readthedocs.io/en/latest/user_guide/hopsworks/featurestore.html#connecting-from-databricks-notebooks>`_.
+Documentation for accessing Hopsworks Featurestore from Databricks Platform is available here: SageMaker_.
```

Comment:
@steffengr sagemaker link in databricks section and vice versa

Reply:
Thanks I fixed that